Source/Scene/ModelExperimental/ModelExperimental.js

```diff
@@ -212,10 +216,31 @@ export default function ModelExperimental(options) {
   // Keeps track of resources that need to be destroyed when the Model is destroyed.
   this._modelResources = [];
 
-  // Computation of the model's bounding sphere and its initial radius is done in ModelExperimentalSceneGraph
+  // Computation of the model's bounding sphere and its initial radius is done
```

Comment:
```suggestion
  // Computation of the model's bounding sphere and its initial radius are done
```

Reply:
Since the verb refers to the noun "computation" it really should be "is" 😛 but I'll edit this so it reads better

Reply:
aaah you're right, I misread it :sweat_smile: but yeah maybe a different wording would help